libjolie/src/main/java/jolie/lang/CodeCheckMessage.java

```diff
@@ -77,7 +77,13 @@ public String toString() {
 			} else {
 				messageBuilder.append( "No descriptive error message found.\n" );
 			}
-			messageBuilder.append( myContext.line() ).append( ':' ).append( myContext.lineString() ).append( '\n' );
+			if( myContext.lineString().contains( myContext.line() + ":" ) ) {
```

Comment:
`lineString` should probably be called differently since it's not a line anymore.